Dockerfile

```diff
@@ -13,9 +14,9 @@ RUN apt-get install -yy \
 RUN pip install --upgrade pip
 RUN pip install github3.py
 
+RUN git clone https://github.com/muvarov/githubscripts.git /githubscripts
+
 RUN echo "#!/bin/bash \n \
-	git clone https://github.com/muvarov/githubscripts.git \n \
```

Comment:
Idea here was that I can update githubscripts.git and changes will be visible on that server with cron update.

Reply:
We need to exec cron_job.sh at build time then, either way the scripts should be in the image at build time.